client/src/components/tree-select/component.tsx

```diff
@@ -69,6 +69,10 @@ const CustomCheckbox = React.forwardRef<
     (ref as RefObject<HTMLInputElement>).current.indeterminate = !checked && indeterminate;
   }, [checked, indeterminate, ref]);
 
+  const onChange = useCallback(() => {
+    // noop
+  }, []);
+
```

Comment:
@JoaquinTrinanes is this ok?

Reply:
Yes. It's kinda awkward, but Next throws a red, big, annoying warning every time there's a checkbox with the `checked` prop set to something but no `onChange` handler. And in tree selects there are a LOT of checkboxes :sweat_smile: